plugins/api/src/atomic_apply_acl_operations.cpp

```diff
@@ -569,17 +569,22 @@ auto plugin_factory(const std::string& _instance_name,
     irods::client_api_allowlist::add(ATOMIC_APPLY_ACL_OPERATIONS_APN);
 #endif // RODS_SERVER
 
+    const auto clear_bbuf_adapter = [](void* _p) { clearBBuf(static_cast<BytesBuf*>(_p)); };
```

Comment:
I feel there's a library here ...

Given a callable, return a new callable matching the supplied signature.

Is this possible to implement?

Reply:
Done